src/ExcelReader.cs

```diff
@@ -46,84 +21,7 @@ public ExcelReader(String filename)
-                            {
-                                throw;
-                            }
-
-                            DataTable schemaTable = conn.GetOleDbSchemaTable(OleDbSchemaGuid.Tables, new object[] { null, null, null, "TABLE" });
-
-                            foreach (DataRow schemaRow in schemaTable.Rows)
-                            {
-                                string sheet = schemaRow["TABLE_NAME"].ToString();
-
-                                if (!sheet.EndsWith("_") && sheet.Contains("$"))
-                                {
-                                    try
-                                    {
-                                        OleDbCommand cmd = new OleDbCommand("SELECT * FROM [" + sheet + "]", conn);
-                                        cmd.CommandType = CommandType.Text;
-
-                                        DataTable outputTable = new DataTable(sheet);
-                                        ds.Tables.Add(outputTable);
-                                        new OleDbDataAdapter(cmd).Fill(outputTable);
-                                        outputTable.Dispose();
-                                    }
-                                    catch (Exception ex)
-                                    {
-                                        throw new Exception(ex.Message + string.Format("Sheet: {0}.File.F{1}", sheet, Filename), ex);
-                                    }
-                                }
-                            }
-
-                            conn.Close();
-                            conn.Dispose();
-                        }
-
-                        ExcelSet = ds;
-
-                        foreach (DataTable dt in ExcelSet.Tables)
-                        {
-                            List<DataRow> deleteRows = new List<DataRow>();
-                            dt.TableName = dt.TableName.Replace("$", String.Empty);
-                            foreach (DataRow row in dt.Rows)
-                            {
-                                bool HasValue = false;
-                                foreach (DataColumn c in dt.Columns)
-                                {
-                                    string value = row[c].ToString();
-                                    if (!String.IsNullOrWhiteSpace(value) || !String.IsNullOrEmpty(value))
-                                    {
-                                        HasValue = true;
-                                        break;
-                                    }
-                                }
-                                if (HasValue == false)
-                                {
-                                    deleteRows.Add(row);
-                                }
-                            }
-                            foreach (DataRow row in deleteRows)
-                            {
-                                dt.Rows.Remove(row);
```

Comment:
Looks like this is missing in a new code, so maybe we should keep it as there is no reason to have empty rows as it will lead to the errors at the import